grass7/raster/r.centroids/r.centroids.html

```diff
@@ -0,0 +1,47 @@
+<h2>DESCRIPTION</h2>
+
+<em>r.centroids</em> is a wrapper function for <a href="https://grass.osgeo.org/grass-stable/manuals/r.volume.html">r.volume</a>
+that computes the center point of raster clumps of data (neighboring, same-valued pixels). The centroids will always fall within
+the clump so they may not be the true, mathematical centroid. The centroids will always fall at the center of a pixel.
+
+
+<h3>Options and flags:</h3>
+
+<em>r.centroids</em> requires an input raster containing clumps of data such as the 
+output of <a href="https://grass.osgeo.org/grass-stable/manuals/r.clump.html">r.clump</a>.
+
+<h2>EXAMPLES</h2>
+Find the centroids of the basins map (North Carolina sample dataset):<br>
+
+<div class="code"><pre>
+#Set computational area
+g.region raster=basin_50K@PERMANENT
+
+#Compute centroids
+r.centroids input=basin_50K@PERMANENT output=centroids50K
```

Comment:
You can also leave out `@PERMANENT`. The rasters are found in the PERMANENT mapset automatically as GRASS is going through the search path looking for the raster (by default the search path is the current mapset and the PERMANENT mapset in this order). Additionally, here we don't have to emphasize from which mapset the raster is coming as the example is focusing on something else.